packages/components/src/components/toggle/stories/index.stories.tsx

```diff
@@ -36,3 +36,18 @@ export const On: React.FC = () => {
     />
   );
 };
+
+export const HiddenLabelText: React.FC = () => {
```

Comment:
Nit: With FC getting depreciated we should avoid using that.

Reply:
@RohitRajendran yeah. I keep forgetting about that.